tdc/dataset_configs/config_map.py

```diff
@@ -0,0 +1,21 @@
+from .pentelute_mdm2_ace2_12ca5_config import PenteluteProteinPeptideConfig
+from .scperturb_config import SCPerturb
+
+scperturb_datasets = [
+    "scperturb_drug_AissaBenevolenskaya2021",
+    "scperturb_drug_SrivatsanTrapnell2020_sciplex2",
+    "scperturb_drug_SrivatsanTrapnell2020_sciplex3",
+    "scperturb_drug_SrivatsanTrapnell2020_sciplex4",
+    "scperturb_drug_ZhaoSims2021",
+]
```

Comment:
@amva13 – how can I use this to load other datasets from scPerturb ([full list](https://www.sanderlab.org/scPerturb/datavzrd/scPerturb_vzrd_v2/dataset_info/index_1.html))? At least I'm interested in using `NormanWeissman2019`, `ReplogleWeissman2022` datasets. I tried to review the codes but I didn't understand this is taking the data directly from scPerturb or these are uploaded elsewhere.

Reply:
@abearab yep you're right this is only the drug-based perturbations. We're building a model on these. We have yet to upload crispr-based perturbations. 

If this is urgent on your end, I can help you make the necessary changes and I can upload the data to Harvard dataverse. I implemented this in such a way the loader should not require changes to support crispr based perturbation datasets in scperturb. We should only need to add the names in the metadata files being used.

Reply:
cc @kexinhuang12345 on this too

Reply:
Yeah i agree that it should be pretty similar format as drug based perturbations. and i don't think there is an easy way to load other datasets directly without first porting them to TDC dataverse.

Reply:
@amva13, @kexinhuang12345 – cool, thank you so much for your response. It would be great to have these datasets and I'll use them immediately. I already started using scperturb but I would prefer having everything through TDC. I created #239 issue to follow this discussion.